formica.go

```diff
@@ -1,3 +0,0 @@
-package main
```

Comment:
lol. i adapted myself the other way around. already started using `main.go` in moa/onsho too :)

so no strong feelings. we should finally decide and put this in the coding guidelines.

https://github.com/giantswarm/onsho/blob/master/main.go